Exercices/JS05/index.js

```diff
@@ -0,0 +1,9 @@
+"use strict"; 
+alert("Enter two numbers to see if the first is greater than the second");
+let fristNumber = prompt("Insert frist number:");
+let secondNumber = prompt("Insert second number:");
```

Comment:
Need to convert prompt values to "number" type before comparing them. Otherwise you're comparing not numbers but strings.